### PR TITLE
Clean up the coverage code a bit

### PR DESCRIPTION
**Pull request by srinivashegde86**

### CODE REVIEW

File: tools/coverage/calc/base.go [hunk 1]
```diff
@@ -48,26 +48,6 @@ func (blk *codeBlock) addToFileCov(coverage *Coverage) {
...
-			isConcerned = false
-			return
-		}
-		isConcerned = !git.IsCoverageSkipped(filePath)
-		(*concernedFiles)[filePath] = isConcerned
-	}
-	return
-}
-
 // add blk Coverage to file group Coverage; return true if the row is concerned
```
Comment (adrcunha) [inline tools/coverage/calc/base.go]: Funny, this is exactly the same comment used in the function you moved. @steuhs what are the right comment for these functions?

File: tools/coverage/calc/base.go [hunk 2]
```diff
@@ -79,6 +59,26 @@ func (blk *codeBlock) addToGroupCov(g *CoverageList) (isConcerned bool) {
 	return true
 }
 
+// add blk Coverage to file group Coverage; return true if the row is concerned
```
Comment (adrcunha) [inline tools/coverage/calc/base.go]: Funny, this is exactly the same comment used in `addToGroupCov`. @steuhs what's the right comment for this function?

File: tools/coverage/calc/delta.go [hunk 3]
```diff
@@ -113,74 +109,69 @@ func NewGroupChanges(baseList *CoverageList, newList *CoverageList) *GroupChange
...
-func (changes *GroupChanges) processChangedFiles(
-	githubFilePaths *map[string]bool, rows *[]string, isEmpty,
-	isCoverageLow *bool) {
-	log.Printf("\nFinding joining set of changed files from profile[count=%d"+
-		"] & github\n", len(changes.Changed))
-	covThres := changes.NewGroup.covThresholdInt
+func (changes *GroupChanges) processChangedFiles(githubFilePaths map[string]bool) (string, bool, bool) {
+	log.Printf("\nFinding joining set of changed files from profile[count=%d] & github\n", len(changes.Changed))
+	rows := []string{
+		"The following is the coverage report on pkg/.",
```
Comment (adrcunha) [inline tools/coverage/calc/delta.go]:
```suggestion
		"The following is the coverage report on the affected files.",
```

**Comment by steuhs** [comment]
Have the changes been deployed and tested?

File: tools/coverage/calc/base.go (hunk 1 above)
Comment (steuhs) [reply tools/coverage/calc/base.go]: updated

**Comment by srinivashegde86** [comment]
> Have the changes been deployed and tested?
Yes - https://prow.knative.dev/view/gcs/knative-prow/pr-logs/pull/knative_serving/5663/pull-knative-serving-go-coverage-dev/1176630126234832896

**Pull request merged**